enterprise/server/configure.md

```diff
@@ -1,9 +1,36 @@
 ---
 layout: default
 title: Configure server-side components
+description: Configuration options for premium server-side components.
 keywords: enterprise tinymcespellchecker spell check checker pro pricing imagetools server configuration configure
 ---
 
+## Overview
+
+* Creating a configuration file
+* General service settings
+  * `allowed-origins` (required)
+    * Wildcard support
+    * Regular Expressions support
+    * `allowed-origins.same-origin` (optional)
+    * `allowed-origins.ignore-port` (optional)
+    * Troubleshooting Origins
+  * `proxy` (optional)
+  * `http` (optional)
+    * Alternative http timeout settings
+* Image proxy service settings
+  * `image-proxy` (optional)
+* Hyperlinking service settings
+  * `link-checking` (optional)
+    * `enabled` (optional)
+    * `fallback-to-get` (optional)
+    * `cache` (optional)
+* Spelling service settings
+  * `spelling` (optional)
+    * `custom-dictionaries-path` (optional)
```

Comment:
Should hunspell be mentioned here in the Overview ? 

- [ ] confirm Hunspell shouldn't be in the overview.

Reply:
Oh, I was going to make those links, seems I forgot. :facepalm:

Reply:
fixed in [`c367b42` (#2048)](https://github.com/tinymce/tinymce-docs/pull/2048/commits/c367b42164d11a3902f0b6d6cc4e0226b58a2d96)